src/context/contextview.cpp

```diff
@@ -631,7 +631,11 @@ void ContextView::UpdateLyrics(const quint64 id, const QString &provider, const
 
   if (static_cast<qint64>(id) != lyrics_id_) return;
 
-  lyrics_ = lyrics + "\n\n(Lyrics from " + provider + ")\n";
+  if (lyrics == nullptr) {
```

Comment:
```suggestion
  if (lyrics.isEmpty()) {
```

Reply:
`lyrics` is not a pointer, but a reference so `nullptr` should not be used, `QString::isEmpty()` covers both empty and null strings.